Source/core/inspector/CodeGeneratorInstrumentation.py

```diff
@@ -125,7 +125,11 @@ class InstrumentingAgents : public RefCountedWillBeGarbageCollectedFinalized<Ins
 public:
     static PassRefPtrWillBeRawPtr<InstrumentingAgents> create()
     {
+#ifndef DISABLE_INSPECTOR
         return adoptRefWillBeNoop(new InstrumentingAgents());
+#else
+        return adoptRefWillBeNoop((InstrumentingAgents*)nullptr);
```

Comment:
I didn't investigate the real reason, but without it, there is a compiling error.